website/shortlink/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.core import validators
+from django.db import models
+
+from django.utils.translation import gettext_lazy as _
+
+
+class ShortLink(models.Model):
+    key = models.CharField(
+        primary_key=True,
+        max_length=32,
+        validators=[validators.validate_slug],
+        verbose_name=_("Name for the url"),
+        help_text=_(f"This is what you use after https://{settings.SITE_DOMAIN}/"),
+    )
```

Comment:
Since this value is used as a primary key it is impossible to change it. I think it would be better to just force it to be unique, but not use it as pk.

Reply:
Good point, I think it would be that bad if we had no primary key here?